caom2pipe/tests/test_run_composable.py

```diff
@@ -1128,15 +1050,23 @@ def test_run_ingest(
         test_config.collection = 'CFHT'
         test_config.proxy_file_name = 'cadcproxy.pem'
         test_config.proxy_fqn = f'{tmp_dir_name}/cadcproxy.pem'
-        test_config.features.supports_latest_client = True
         test_config.use_local_files = False
+        test_config.log_file_directory = tmp_dir_name
```

Comment:
This seems to repeat throughout the file. Could it be factored out in a separate function?

Reply:
I added the method `change_working_directory` to the `manage_composable.Config` class implementation, and shifted to using it along with the `tmpdir` fixture. Does that address your concerns?